Tests_WebCompiler/MinifyJsTests.cs

```diff
@@ -13,7 +13,7 @@ public void CreatePipeline()
             pipeline = (file) => new CompilationStep(file).With(new JavascriptMinifier(new JavaScriptMinifySettings { TermSemicolons = false }));
             input = "../../../TestCases/Js/test.js";
             output_files = new List<string> { "../../../TestCases/Js/test.min.js" };
-            expected_output = "../../../TestCases/MinJs/test.min.js";
+            expected_output = "../../../TestCases/Js/test.min.js";
```

Comment:
See comments on #10

Reply:
Only now did I notice that the file was missing in the repo... My global gitignore made me miss that one. Sorry about that!

Reply:
It happens! At least you didn't accidentally commit SSH keys 😁